bal_tools/subgraph.py

```diff
@@ -453,6 +453,43 @@ def get_balancer_pool_snapshots(
                 break
         return all_pools
 
+
+    def query_pools_by_tvl_and_swapfee(self,
+                                lower_boundry=0.0001,
+                                upper_boundry=0.0003,
+                                min_tvl=500_000,
+                                pool_types=["STABLE", "META_STABLE", "COMPOSABLE_STABLE"],
+                                chains=None):
+        if chains is None:
+            chains = [self.chain.upper()]
+        print(f"Running on {chains}")
+        where = {"where":
+            {
+                "minTvl": min_tvl,
+                "poolTypeIn": pool_types,
+                "chainIn": chains
+            }
+        }
+        print(where)
+        results = {}
+        query = self.fetch_graphql_data("apiv3", "get_swap_fees", where)
+        for result in query["poolGetPools"]:
+            if float(result["dynamicData"]["swapFee"]) < lower_boundry or float(result["dynamicData"]["swapFee"]) > upper_boundry:
```

Comment:
so if the swap fee is lower than the lower boundry or higher than the upper boundry add the pool to the final results? should it not be if the swap fee falls between the lower and upper bounds?

Reply:
Will take a bit more time, allow filtering/output of TVL and Volume too, and add some tests. 

BalPoolsGauges seems like a good place for it.

Reply:
The point here is to look for outliers that sit outside a range.  Maybe I need to work on the function name/docs.